### ISSUE TITLE
OpenCV Error: Unspecified error (Input layer not found: conv/batchnorm) in connect, file 

### ISSUE THREAD

**Issue by AndyLin0128** [opencv/opencv#9556]
OpenCV Error: Unspecified error (Input layer not found: conv/batchnorm) in connect, file /tmp/opencv3-20170811-60097-12ytr4g/opencv-3.3.0/modules/dnn/src/tensorflow/tf_importer.cpp, line 447
libc++abi.dylib: terminating with uncaught exception of type cv::Exception: /tmp/opencv3-20170811-60097-12ytr4g/opencv-3.3.0/modules/dnn/src/tensorflow/tf_importer.cpp:447: error: (-2) Input layer not found: conv/batchnorm in function connect

**Comment by dkurt** [comment]
@AndyLin0128, it's good to know at least the model you used.

**Comment by cyberguy42** [comment]
I encountered a very similar error a few days ago. By stepping through with a debugger, I discovered that the problem was that the split operator did not accept constants as an input. There are a few other TODO's mentioned in the source comments; you may want to read through those and confirm that your graph doesn't require any of the missing capabilities.

**Comment by dkurt** [comment]
@cyberguy42, could you reference to the model or provide a minimal working example?
